### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jtymoszczuk**
Added a little more clarification for users who are new to command line to run it locally.

### CODE REVIEW

**Comment by ecurrencyhodler** [comment]
Wow thanks for submitting a pr joe! 

LGTM. But I'd like a dev to review this before merging.

File: README.md [hunk 1]
```diff
@@ -38,13 +38,13 @@ To help with translations and review head over to our translation [README.md](ht
...
 
-1. First download the code from this repository
-2. Ensure you have NodeJS version 16.13.0 or higher installed
-3. Run `npm install` to download the required code libraries
-4. Run `npm run build` to build
-5. Run `npm run start` to spin up the development environment
-6. Open `http://localhost:3000` in your browser to access the site
-
+1. First, make sure you have NodeJS version 16.13.0 or higher installed. Check `node -v`. Node [installation instructions](https://nodejs.org/en/download/package-manager/)
+3. Download this repository `git clone https://github.com/saving-satoshi/saving-satoshi.git`
```
Comment (vinayaksh42) [inline README.md:42]: I think you skipped number `2`

**Review by vinayaksh42 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jtymoszczuk) [reply README.md:42]: Hmmmm, when I click the readme doc and look at the other view the '2.' Is there.

**Review by shaavan — APPROVED** [review]
ACK

- Added link is correct.
- Instructions are coherent.

**Review by ecurrencyhodler — APPROVED** [review]

**Pull request merged**